src/or/nodelist.h

```diff
@@ -29,6 +29,7 @@ const node_t *node_get_by_hex_id(const char *identity_digest,
 node_t *nodelist_set_routerinfo(routerinfo_t *ri, routerinfo_t **ri_old_out);
 node_t *nodelist_add_microdesc(microdesc_t *md);
 void nodelist_set_consensus(networkstatus_t *ns);
+void ensure_nodelist_freshness(void);
```

Comment:
I propose we namespace this one starting with `nodelist_`.

Reply:
Good point. Done in aa951c72dc.

Reply:
ACK.